easybuild/easyconfigs/d/DAS_Tool/DAS_Tool-1.1.1-foss-2021a-R-4.1.0.eb

```diff
@@ -0,0 +1,59 @@
+easyblock = 'Tarball'
+
+name = 'DAS_Tool'
+version = '1.1.1'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://github.com/cmks/DAS_Tool'
+description = """DAS Tool is an automated method that integrates the results of a flexible number of binning
+ algorithms to calculate an optimized, non-redundant set of bins from a single assembly."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = ['https://github.com/cmks/DAS_Tool/archive/']
+sources = ['%(version)s.tar.gz']
+patches = ['DAS_Tool-1.1.1-deafaultSearchEngBLAST.patch']
+checksums = [
+    '2a55f67b5331251d8fd5adea867cc341363fbf7fa7ed5c3ce9c7679d8039f03a',  # 1.1.1.tar.gz
+    'dc89e6ab7129d170d24da28720cec0937b4d958b5ae46049373d57dde8b8a27a',  # DAS_Tool-1.1.1-deafaultSearchEngBLAST.patch
+]
+
+dependencies = [
+    ('R', '4.1.0'),
+    ('Python', '3.9.5'),
+    ('Ruby', '3.0.1'),
+    ('pullseq', '1.0.2'),
+    ('prodigal', '2.6.3'),
+    ('BLAST+', '2.11.0'),
+]
+
+exts_defaultclass = 'RPackage'
+
+exts_list = [
+    (name, version, {
+        'modulename': 'DASTool',
+        'source_tmpl': 'DASTool_%(version)s.tar.gz',
```

Comment:
@smoretti I guess this should be changed to `%(version)s.tar.gz` too so it works for `1.1.5`?

I'm a bit confused why this works for `1.1.1`...

Reply:
No, this link (line 35) does not work for version 1.1.5

So maybe we have to stay with version 1.1.1

Reply:
Everything looks fine with version 1.1.3